src/main/java/net/backupcup/mcde/screen/handler/RollBenchScreenHandler.java

```diff
@@ -1,40 +1,60 @@
 package net.backupcup.mcde.screen.handler;
 
+import java.util.EnumMap;
 import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
 
 import net.backupcup.mcde.MCDEnchantments;
 import net.backupcup.mcde.block.ModBlocks;
 import net.backupcup.mcde.util.EnchantmentSlots;
 import net.backupcup.mcde.util.EnchantmentUtils;
 import net.backupcup.mcde.util.Slots;
+import net.fabricmc.fabric.api.networking.v1.PacketByteBufs;
+import net.fabricmc.fabric.api.networking.v1.PacketSender;
+import net.fabricmc.fabric.api.networking.v1.ServerPlayNetworking;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.network.ClientPlayNetworkHandler;
 import net.minecraft.enchantment.EnchantmentHelper;
 import net.minecraft.entity.player.PlayerEntity;
 import net.minecraft.entity.player.PlayerInventory;
 import net.minecraft.inventory.Inventory;
 import net.minecraft.inventory.SimpleInventory;
 import net.minecraft.item.ItemStack;
 import net.minecraft.item.Items;
+import net.minecraft.network.PacketByteBuf;
 import net.minecraft.screen.ScreenHandler;
 import net.minecraft.screen.ScreenHandlerContext;
+import net.minecraft.screen.ScreenHandlerListener;
 import net.minecraft.screen.slot.Slot;
+import net.minecraft.server.network.ServerPlayerEntity;
 import net.minecraft.sound.SoundCategory;
 import net.minecraft.sound.SoundEvents;
 import net.minecraft.util.Identifier;
 
-public class RollBenchScreenHandler extends ScreenHandler {
+public class RollBenchScreenHandler extends ScreenHandler implements ScreenHandlerListener {
     private final Inventory inventory = new SimpleInventory(2);
     private final ScreenHandlerContext context;
+    private final PlayerEntity player;
```

Comment:
In Yarn mappings, it is common to have the assigned name match the class that the object is from. Therefore, this should be:
`private final PlayerEntity playerEntity`
Whilst this is not necessary, it allows for a greater level of consistency across one's code whilst also allowing for those who might be looking at your code to be able to read it in a manner with which they are accustomed.